src/stars.css

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
you should add all styles from prev task for stars. If I change stars count in classes I want to see right active stars count